eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -165,6 +165,18 @@
         "code": "java.method.finalMethodAddedToNonFinalClass",
         "new": "method <V> reactor.core.publisher.Flux<V> reactor.core.publisher.Flux<T>::switchOnFirst(java.util.function.BiFunction<reactor.core.publisher.Signal<? extends T>, reactor.core.publisher.Flux<T>, org.reactivestreams.Publisher<? extends V>>, boolean) @ com.azure.core.util.paging.ContinuablePagedFluxCore<C, T, P extends com.azure.core.util.paging.ContinuablePage<C extends java.lang.Object, T extends java.lang.Object>>",
         "justification": "Reactor added new final APIs to Flux."
+      },
+      {
+        "regex": true,
```

Comment:
@JonathanGiles @srnagar mind taking a look at this change. Models classes are having new annotations and new annotation attributes due to codegen being used instead of the hand-written layer. These annotation changes better represent how the model class should behave but may lead to functional difference.

Reply:
As discussed this is specific to Search, as of right now (and any newer SDKs will begin with this functionality). We are okay with making this change for Search as it is newly released but changes such as this in the future must be reviewed.